.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       # Setup .npmrc file to publish to GitHub Packages
       - uses: actions/setup-node@v3
         with:
-          registry-url: 'https://npm.pkg.github.com'
+          registry-url: 'https://registry.npmjs.org'
```

Comment:
reason for this change detailed [here](https://github.com/scientist-softserv/webstore-component-library/pull/231/commits/812a0424171e80357e32ab9707985c715c2bc469).

Reply:
thread that led me to this change: https://github.com/npm/cli/issues/6184#issuecomment-1544496444

Reply:
`0.1.24` was also successfully registered on [npmjs](https://www.npmjs.com/package/@scientist-softserv/webstore-component-library?activeTab=versions).